Jenkinsfile

```diff
@@ -58,17 +62,25 @@ Map branchJobs(String[] branches) {
 pipeline {
     agent none
 
+    options {
+        skipDefaultCheckout()
+    }
+
     stages {
-        stage('Versions') {
+        stage('Read version information') {
             agent {
                 label 'docker'
             }
             steps {
+                checkoutCommunityRepoBranch(
+                    'rticonnextdds-cmake-utils', env.BRANCH_NAME
+                )
                 script {
-                    String[] examplesBranches = readYaml(
-                        file: "${env.WORKSPACE}/resources/ci/config.yaml"
-                    ).versions.keySet()
-                    parallel branchJobs(examplesBranches)
+                    CONNEXT_VERSIONS.addAll(
```

Comment:
CONNEXT_VERSIONS contains the branches to build, right?

Reply:
Yes!